.github/scripts/install-bitcoind.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -x
```

Comment:
Change this to:

```
#! /bin/sh

set -e
```

There are no bashisms in here, and you really do want to fail if anything fails!